ign_ros2_control_demos/urdf/test_cart_velocity.xacro.urdf

```diff
@@ -123,6 +123,7 @@
       <state_interface name="position"/>
       <state_interface name="velocity"/>
       <state_interface name="effort"/>
+      <param name="p_vel">100.0</param>
```

Comment:
Why this change? Seems unrelated to the gripper functionality of this PR

Reply:
It is not, as it uses velocity command interface which is mapped using PID to the `ForceJointCmd`